Packages/com.chisel.editor/Chisel/Editor/Editor/Scene/CSGSceneBottomGUI.cs

```diff
@@ -26,6 +26,11 @@ public static class CSGSceneBottomGUI
 		static GUIStyle buttonStyle;
 		static bool     prevSkin = false;
 
+		// UI Element definitions
+		static GUIContent rebuildButton = new GUIContent("Rebuild");
+		static GUIContent doubleSnapDistanceButton = new GUIContent("+", "Double the snapping distance.\nHotkey: ]");
+		static GUIContent halveSnapDistanceButton = new GUIContent("-", "Halve the snapping distance.\nHotkey: [");
+
```

Comment:
This is where custom tooltips would be nice, it allows us to write a system where hotkeys are updated automatically